pkg/ccl/streamingccl/streamingest/datadriven_test.go

```
@@ -378,30 +390,43 @@ func (d *datadrivenTestState) cleanup(t *testing.T) {
 	}
 }
 
-func (d *datadrivenTestState) queryAs(t *testing.T, as, query string) string {
+func (d *datadrivenTestState) queryAs(
+	ctx context.Context, t *testing.T, as, query string,
+) (string, error) {
 	var rows *gosql.Rows
+	var err error
 	switch as {
 	case "source-system":
-		rows = d.replicationClusters.SrcSysSQL.Query(t, query)
+		rows, err = d.replicationClusters.SrcSysSQL.DB.QueryContext(ctx, query)
 	case "source-tenant":
-		rows = d.replicationClusters.SrcTenantSQL.Query(t, query)
+		rows, err = d.replicationClusters.SrcTenantSQL.DB.QueryContext(ctx, query)
 	case "destination-system":
-		rows = d.replicationClusters.DestSysSQL.Query(t, query)
+		rows, err = d.replicationClusters.DestSysSQL.DB.QueryContext(ctx, query)
 	case "destination-tenant":
-		rows = d.replicationClusters.DestTenantSQL.Query(t, query)
+		rows, err = d.replicationClusters.DestTenantSQL.DB.QueryContext(ctx, query)
 	default:
 		t.Fatalf("unsupported value to run SQL query as: %s", as)
 	}
 
+	if err != nil {
+		return "", err
+	}
 	output, err := sqlutils.RowsToDataDrivenOutput(rows)
 	require.NoError(t, err)
```

Should we return this error now that we have an error return?

---

As is works for me.